dashboard/api.py

```diff
@@ -138,18 +136,29 @@ def get_info_for_program(program, user, enrollments, certificates):
     }
     for course in program.course_set.all():
         data['courses'].append(
-            get_info_for_course(user, course, enrollments, certificates)
+            get_info_for_course(course, enrollments, certificates)
         )
     data['courses'].sort(key=lambda x: x['position_in_program'])
     return data
 
 
-def get_info_for_course(user, course, user_enrollments, user_certificates):
+def _add_run(course_data, next_run, status, certificate=None):
```

Comment:
I still feel like this parameter should be called `run`, and not `next_run`, what do you think?